lib/begin.go

```diff
@@ -80,17 +82,59 @@ func (a *ACBuild) Begin(start string, insecure bool) (err error) {
 			return err
 		}
 		if start[0] == '.' || start[0] == '/' {
-			return a.beginFromLocalImage(start)
+			finfo, err := os.Stat(start)
+			switch {
+			case os.IsNotExist(err):
+				return fmt.Errorf("no such file or directory: %s", start)
+			case err != nil:
+				return err
+			case finfo.IsDir():
+				a.beginFromLocalDirectory(start)
+			default:
+				return a.beginFromLocalImage(start)
+			}
 		} else {
 			return a.beginFromRemoteImage(start, insecure)
 		}
 	}
+	return a.beginWithEmptyACI()
+}
+
+func (a *ACBuild) beginFromLocalImage(start string) error {
+	finfo, err := os.Stat(start)
+	if err == nil {
```

Comment:
this branching is slightly awkward, `if err != nil { return err }` seems more natural